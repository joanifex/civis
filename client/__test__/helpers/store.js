import configureMockStore from 'redux-mock-store';


const store = (model) => {
  const mockStore = configureMockStore();
  return mockStore(model)
}

export default store;
