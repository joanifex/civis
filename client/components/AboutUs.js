import React from 'react';

class AboutUs extends React.Component{
  render (){
    return(
      <div className='row'>
        <h1 className='center'>About Us</h1>
        <div className='col s12 m6 l3'>
          <h5 className='center'>Ian</h5>
          <br />
          <img width='100%' src='https://s-media-cache-ak0.pinimg.com/564x/10/3f/62/103f62d4e3aa8f471b3b512c9361cfe1.jpg' />
          <p>Hi, I am Ian and I'm the smartest person in the class! </p>
        </div>
        <div className='col s12 m6 l3'>
          <h5 className='center'>Jeremy</h5>
          <br />
          <img width='100%' src='http://scontent.cdninstagram.com/t51.2885-15/s480x480/e35/13531800_1235678766445299_1590304831_n.jpg?ig_cache_key=MTI5MDUzMDU5NjcwNzI0MzQ3Mw%3D%3D.2'/>
          <p>Hi! I am Jeremy and I'm a lumberjack and I'm ok! </p>
        </div>
        <div className='col s12 m6 l3'>
          <h5 className='center'>Lindsay</h5>
          <br />
          <img width='100%' src='https://img.buzzfeed.com/buzzfeed-static/static/enhanced/webdr01/2013/4/18/16/anigif_enhanced-buzz-1786-1366317339-16_preview.gif' />
          <p>Hi! I'm Lindsay and I like drinks! How much could a banana cost? $10?! </p>
        </div>
        <div className='col s12 m6 l3'>
          <h5 className='center'>Valerie</h5>
          <br />
          <img width='100%' src='http://i.dailymail.co.uk/i/pix/2015/09/04/13/190E8A9100000578-0-image-a-26_1441371211282.jpg'/>
          <p>Hi! My name is Valerie and I want to look like Liza</p>
        </div>
      </div>
    );
  }
}

export default AboutUs;
