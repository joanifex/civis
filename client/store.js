import { createStore, compose, applyMiddleware } from 'redux';
import { syncHistoryWithStore } from 'react-router-redux';
import { browserHistory } from 'react-router';
import thunk from 'redux-thunk';

import rootReducer from './reducers/index';

const enhancers = compose(
  applyMiddleware(thunk),
  window.devToolsExtension ? window.devToolsExtension() : f => f
)

let appDataSet = document.getElementById('app').dataset;
let remember = appDataSet.remember;

remember = remember === 'true' ? true : false;

const store = createStore(rootReducer, { auth: { isAuthenticated: remember } }, enhancers)

if(module.hot) {
  module.hot.accept('./reducers/',() => {
    const nextRootReducer = require('./reducers/index').default;
    store.replaceReducer(nextRootReducer);
  });
}

export const history = syncHistoryWithStore(browserHistory, store)

export default store
