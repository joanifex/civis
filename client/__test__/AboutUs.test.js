import React from 'react';
import AboutUs from '../components/AboutUs.js';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from './helpers/store';

test('AboutUs Component is rendered', () => {
  const component = renderer.create(
    <Provider store={store({})}>
      <AboutUs />
   </Provider>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
