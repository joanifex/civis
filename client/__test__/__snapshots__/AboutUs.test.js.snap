// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs Component is rendered 1`] = `
<div
  className="row"
>
  <h1
    className="center"
  >
    Our Team
  </h1>
  <div
    className="col s12 m6 l3"
  >
    <h5
      className="center"
    >
      Ian Wright
    </h5>
    <br />
    <img
      src="https://s-media-cache-ak0.pinimg.com/564x/10/3f/62/103f62d4e3aa8f471b3b512c9361cfe1.jpg"
      width="100%"
    />
    <p>
      Hi, I am Ian and I'm the smartest person in the class! 
    </p>
    <br />
    <a
      href="https://github.com/wrightianb"
      target="_blank"
    >
      <img
        src="http://res.cloudinary.com/dx6ytyvvg/image/upload/v1489866531/github-logo_vpclpu.png"
      />
    </a>
    <a
      href="https://www.linkedin.com/in/wrightianb/"
      target="_blank"
    >
      <img
        src="http://res.cloudinary.com/dx6ytyvvg/image/upload/v1489866614/linkedin-icon.png"
      />
    </a>
    <a
      className="email"
      href="mailto:ian@test.com?subject=Civis"
    >
      <img
        src="http://res.cloudinary.com/dx6ytyvvg/image/upload/v1489866524/email-icon.png"
      />
    </a>
  </div>
  <div
    className="col s12 m6 l3"
  >
    <h5
      className="center"
    >
      Jeremy Cram
    </h5>
    <br />
    <img
      src="http://scontent.cdninstagram.com/t51.2885-15/s480x480/e35/13531800_1235678766445299_1590304831_n.jpg?ig_cache_key=MTI5MDUzMDU5NjcwNzI0MzQ3Mw%3D%3D.2"
      width="100%"
    />
    <p>
      Hi! I am Jeremy and I'm a lumberjack and I'm ok! 
    </p>
    <a
      href="https://github.com/jeremycram"
      target="_blank"
    >
      <img
        src="http://res.cloudinary.com/dx6ytyvvg/image/upload/v1489866531/github-logo_vpclpu.png"
      />
    </a>
    <a
      href="https://www.linkedin.com/in/jeremy-cram/"
      target="_blank"
    >
      <img
        src="http://res.cloudinary.com/dx6ytyvvg/image/upload/v1489866614/linkedin-icon.png"
      />
    </a>
    <a
      className="email"
      href="mailto:jeremy@test.com?subject=Civis"
    >
      <img
        src="http://res.cloudinary.com/dx6ytyvvg/image/upload/v1489866524/email-icon.png"
      />
    </a>
  </div>
  <div
    className="col s12 m6 l3"
  >
    <h5
      className="center"
    >
      Lindsay Larkin
    </h5>
    <br />
    <img
      src="https://img.buzzfeed.com/buzzfeed-static/static/enhanced/webdr01/2013/4/18/16/anigif_enhanced-buzz-1786-1366317339-16_preview.gif"
      width="100%"
    />
    <p>
      Hi! I'm Lindsay and I like drinks! How much could a banana cost? $10?! 
    </p>
    <br />
    <a
      href="https://github.com/lilarkin"
      target="_blank"
    >
      <img
        src="http://res.cloudinary.com/dx6ytyvvg/image/upload/v1489866531/github-logo_vpclpu.png"
      />
    </a>
    <a
      href="https://www.linkedin.com/in/lindsaylarkin/"
      target="_blank"
    >
      <img
        src="http://res.cloudinary.com/dx6ytyvvg/image/upload/v1489866614/linkedin-icon.png"
      />
    </a>
    <a
      className="email"
      href="mailto:lindsay@test.com?subject=Civis"
    >
      <img
        src="http://res.cloudinary.com/dx6ytyvvg/image/upload/v1489866524/email-icon.png"
      />
    </a>
  </div>
  <div
    className="col s12 m6 l3"
  >
    <h5
      className="center"
    >
      Valerie Barela
    </h5>
    <br />
    <img
      src="http://i.dailymail.co.uk/i/pix/2015/09/04/13/190E8A9100000578-0-image-a-26_1441371211282.jpg"
      width="100%"
    />
    <p>
      Hi! My name is Valerie and I want to look like Liza
    </p>
    <br />
    <a
      href="https://github.com/vbarela1"
      target="_blank"
    >
      <img
        src="http://res.cloudinary.com/dx6ytyvvg/image/upload/v1489866531/github-logo_vpclpu.png"
      />
    </a>
    <a
      href="https://www.linkedin.com/in/barelavalerie/"
      target="_blank"
    >
      <img
        src="http://res.cloudinary.com/dx6ytyvvg/image/upload/v1489866614/linkedin-icon.png"
      />
    </a>
    <a
      className="email"
      href="mailto:valerie@test.com?subject=Civis"
    >
      <img
        src="http://res.cloudinary.com/dx6ytyvvg/image/upload/v1489866524/email-icon.png"
      />
    </a>
  </div>
</div>
`;
